rdf4j/src/test/scala/org/w3/banana/rdf4j/Rdf4jSparqlGraphTest.scala

```diff
@@ -0,0 +1,10 @@
+package org.w3.banana.rdf4j
+
+import org.w3.banana._
+import org.w3.banana.io._
+import org.w3.banana.rd4j.Rdf4j
+import org.w3.banana.rd4j.io.Rdf4jSolutionsWriter
+
+class Rdf4jSparqlGraphTestJson(implicit sparqlWriter: SparqlSolutionsWriter[Rdf4j, SparqlAnswerJson] = Rdf4jSolutionsWriter.solutionsWriterJson) extends SparqlGraphTest[Rdf4j, SparqlAnswerJson]
```

Comment:
I had problems with this implicit. I remember have had this in real life projects too, exclusively with the SparqlSolutionWriter. Is this because it is a type and not a trait? Cannot see any difference compared to the other implementations.